bus/bus.go

```diff
@@ -0,0 +1,102 @@
+//
+// Copyright 2019 Insolar Technologies GbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package bus
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/ThreeDotsLabs/watermill/message"
+	"github.com/ThreeDotsLabs/watermill/message/router/middleware"
+	"github.com/insolar/insolar/instrumentation/inslogger"
+)
+
+const ExternalMsgTopic = "ExternalMsgTopic"
```

Comment:
Topic string probably should not contain "topic" word. Will "outcoming"/"incoming" suffice?